tests/data/cases/preview_remove_lone_list_item_parens.py

```diff
@@ -0,0 +1,156 @@
+# flags: --unstable
+items = [(123)]
```

Comment:
Add a test containing a generator expression, to make sure we don't turn it into a listcomp:

```
>>> [(x for x in [1])]
[<generator object <genexpr> at 0x104277400>]
>>> [x for x in [1]]
[1]
```

Also add tests with comments in various places (e.g., after the opening paren, before the closing paren). Such cases tend to expose bugs in Black.